Ai-security-best-practices-intermediate.md

```
@@ -0,0 +1,878 @@
+    style L5 fill:#1a2f4a,color:#fff
+```
+
+この全体像を踏まえたうえで、業界標準のフレームワークがそれぞれの層をどうカバーしているかを次章で整理します。
+
+## 3. 主要セキュリティフレームワークの理解
+
+AIセキュリティには「唯一の標準」は存在せず、複数のフレームワークを組み合わせて使うのが実務上の標準的アプローチです。それぞれの役割分担を理解することが最初のステップです。
+
+| フレームワーク | 発行元 | 性質 | 主なカバー範囲 | 最新状況（2026年7月時点） |
+|---|---|---|---|---|
+| OWASP Top 10 for LLM Applications | OWASP GenAI Security Project | 優先度付きリスクリスト（チェックリスト型） | プロンプトインジェクション、機密情報漏洩、サプライチェーン、データ/モデルポイズニング等10項目 | 2025年版が最新。LLM01〜LLM10として体系化 [1][3] |
```

markdownlint-cli2 (0.22.1)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "3"

(MD052, reference-links-images)